### ISSUE TITLE
Comment box not shrinking on submit

### ISSUE THREAD

**Issue by Link2Twenty** [forem/forem#13298]
**Describe the bug**
Comment boxes grow as you type but when you press submit they stay the size they were when they had content.

**To Reproduce**
- Go to a post
- Write a long comment
- Post the comment
- See the comment box is still large.

**Expected behaviour**
I'd expect the comment box the shrink back down to it's original size.

**Screenshots**
![image](https://user-images.githubusercontent.com/3534427/113887114-20cd4700-97b9-11eb-9c14-49c0432ad96f.png)

**Desktop (please complete the following information):**

- OS, version: Window 10
- Browser, version: Chrome 89.0.4389.114

**Additional context**
Another small bug that doesn't really cause any harm.

**Comment by cmgorton** [comment]
I noticed this yesterday too! Thanks for submitting an issue.

**Comment by cmgorton** [comment]
Thanks @oyeyipowale ! I'll assign you to the issue.

**Comment by oyeyipowale** [comment]
Hey @cmgorton, thanks for the response.

I ran into some issues while trying to set up the project using the container (docker) on windows 10. Two compose services are having trouble connecting and I do not know how to debug them.

```bash
sidekiq_1        | 2021/05/02 08:09:46 Problem with request: Get http://rails:3000: dial tcp 172.20.0.2:3000: connect: connection refused. Sleeping 20s
seed_1           | 2021/05/02 08:10:04 Problem with request: Get http://rails:3000: dial tcp 172.20.0.2:3000: connect: connection refused. Sleeping 20s
sidekiq_1        | 2021/05/02 08:10:06 Problem with request: Get http://rails:3000: dial tcp 172.20.0.2:3000: connect: connection refused. Sleeping 20s
seed_1           | 2021/05/02 08:10:24 Problem with request: Get http://rails:3000: dial tcp 172.20.0.2:3000: connect: connection refused. Sleeping 20s
sidekiq_1        | 2021/05/02 08:10:26 Problem with request: Get http://rails:3000: dial tcp 172.20.0.2:3000: connect: connection refused. Sleeping 20s
seed_1           | 2021/05/02 08:10:44 Problem with request: Get http://rails:3000: dial tcp 172.20.0.2:3000: connect: connection refused. Sleeping 20s
sidekiq_1        | 2021/05/02 08:10:46 Problem with request: Get http://rails:3000: dial tcp 172.20.0.2:3000: connect: connection refused. Sleeping 20s
seed_1           | 2021/05/02 08:11:04 Problem with request: Get http://rails:3000: dial tcp 172.20.0.2:3000: connect: connection refused. Sleeping 20s
sidekiq_1        | 2021/05/02 08:11:06 Problem with request: Get http://rails:3000: dial tcp 172.20.0.2:3000: connect: connection refused. Sleeping 20s
seed_1           | 2021/05/02 08:11:24 Problem with request: Get http://rails:3000: dial tcp 172.20.0.2:3000: connect: connection refused. Sleeping 20s
sidekiq_1        | 2021/05/02 08:11:26 Problem with request: Get http://rails:3000: dial tcp 172.20.0.2:3000: connect: connection refused. Sleeping 20s
seed_1           | 2021/05/02 08:11:44 Problem with request: Get http://rails:3000: dial tcp 172.20.0.2:3000: connect: connection refused. Sleeping 20s
sidekiq_1        | 2021/05/02 08:11:46 Problem with request: Get http://rails:3000: dial tcp 172.20.0.2:3000: connect: connection refused. Sleeping 20s
seed_1           | 2021/05/02 08:12:04 Problem with request: Get http://rails:3000: dial tcp 172.20.0.2:3000: connect: connection refused. Sleeping 20s
sidekiq_1        | 2021/05/02 08:12:06 Problem with request: Get http://rails:3000: dial tcp 172.20.0.2:3000: connect: connection refused. Sleeping 20s
seed_1           | 2021/05/02 08:12:24 Problem with request: Get http://rails:3000: dial tcp 172.20.0.2:3000: connect: connection refused. Sleeping 20s
sidekiq_1        | 2021/05/02 08:12:26 Problem with request: Get http://rails:3000: dial tcp 172.20.0.2:3000: connect: connection refused. Sleeping 20s
seed_1           | 2021/05/02 08:12:44 Problem with request: Get http://rails:3000: dial tcp 172.20.0.2:3000: connect: connection refused. Sleeping 20s
sidekiq_1        | 2021/05/02 08:12:46 Problem with request: Get http://rails:3000: dial tcp 172.20.0.2:3000: connect: connection refused. Sleeping 20s
seed_1           | 2021/05/02 08:13:04 Problem with request: Get http://rails:3000: dial tcp 172.20.0.2:3000: connect: connection refused. Sleeping 20s
sidekiq_1        | 2021/05/02 08:13:06 Problem with request: Get http://rails:3000: dial tcp 172.20.0.2:3000: connect: connection refused. Sleeping 20s
seed_1           | 2021/05/02 08:13:24 Problem with request: Get http://rails:3000: dial tcp 172.20.0.2:3000: connect: connection refused. Sleeping 20s
sidekiq_1        | 2021/05/02 08:13:26 Problem with request: Get http://rails:3000: dial tcp 172.20.0.2:3000: connect: connection refused. Sleeping 20s
seed_1           | 2021/05/02 08:13:44 Problem with request: Get http://rails:3000: dial tcp 172.20.0.2:3000: connect: connection refused. Sleeping 20s
sidekiq_1        | 2021/05/02 08:13:46 Problem with request: Get http://rails:3000: dial tcp 172.20.0.2:3000: connect: connection refused. Sleeping 20s
seed_1           | 2021/05/02 08:14:04 Problem with request: Get http://rails:3000: dial tcp 172.20.0.2:3000: connect: connection refused. Sleeping 20s
sidekiq_1        | 2021/05/02 08:14:06 Problem with request: Get http://rails:3000: dial tcp 172.20.0.2:3000: connect: connection refused. Sleeping 20s
seed_1           | 2021/05/02 08:14:24 Problem with request: Get http://rails:3000: dial tcp 172.20.0.2:3000: connect: connection refused. Sleeping 20s
sidekiq_1        | 2021/05/02 08:14:26 Problem with request: Get http://rails:3000: dial tcp 172.20.0.2:3000: connect: connection refused. Sleeping 20s
Gracefully stopping... (press Ctrl+C again to force)
Stopping forem_seed          ... done
Stopping forem_sidekiq       ... done
Stopping forem_webpacker     ... done
Stopping forem_rails         ...
Stopping forem_postgresql    ...
Stopping forem_elasticsearch ...
Stopping forem_redis         ...
Killing forem_rails          ... error
Killing forem_postgresql     ... done
Killing forem_elasticsearch  ... done
Killing forem_redis          ... done

ERROR: for forem_rails  Cannot kill container: 59ee94ed8bf994078a2da68204f5909645aa8e4914e24b7d1240101002e1a9b9: Container 59ee94ed8bf994078a2da68204f5909645aa8e4914e24b7d1240101002e1a9b9 is not running
```
And when I `Ctrl+C` to stop the server, I got the final error above.

Attempting to go to `localhost:3000` just only shows the default react page.

Environment:
- windows 10
- Docker version 19.03.8, build afacb8b
- docker-compose version 1.25.5, build 8a1c60f6

**Comment by Link2Twenty** [comment]
I'd recommend going the WSL route for installation, it's much easier.
 <https://docs.forem.com/installation/windows/>

**Comment by oyeyipowale** [comment]
Thanks, @Link2Twenty for the pointer. I jumped straight to the **container** submenu.
Decide the through with Windows WSL installation and all was great and good. Ran the `bin/setup` command. I got some errors at the command: `bin/rails app_initializer:setup forem:setup`

I tried adding USERNAME and PASSWORD to the DATABASE_URL, got the following error:
```bash
== Preparing database ==
rails aborted!
ActiveRecord::ConnectionNotEstablished: FATAL:  password authentication failed for user "<username>"
/home/user/code/osc/forem/bin/rails:5:in `<top (required)>'
/home/user/code/osc/forem/bin/spring:10:in `require'
/home/user/code/osc/forem/bin/spring:10:in `block in <top (required)>'
/home/user/code/osc/forem/bin/spring:7:in `tap'
/home/user/code/osc/forem/bin/spring:7:in `<top (required)>'

Caused by:
PG::ConnectionBad: FATAL:  password authentication failed for user "<username>"
/home/user/code/osc/forem/bin/rails:5:in `<top (required)>'
/home/user/code/osc/forem/bin/spring:10:in `require'
/home/user/code/osc/forem/bin/spring:10:in `block in <top (required)>'
/home/user/code/osc/forem/bin/spring:7:in `tap'
/home/user/code/osc/forem/bin/spring:7:in `<top (required)>'
Tasks: TOP => db:prepare
(See full trace by running task with --trace)
```

The database URL config in .env file

```
# my edit
DATABASE_URL="postgresql://<username>:<password>@localhost:5432/$DATABASE_NAME"

# default
DATABASE_URL="postgresql://localhost:5432/$DATABASE_NAME"
```

**Comment by Link2Twenty** [comment]
The bit where it says `<username>:<password>` does that contain the actual username and password for the database?

**Comment by oyeyipowale** [comment]
It did contain it. I edited it out before commenting. Both the username and the password.

**Comment by Link2Twenty** [comment]
The error indicates it doesn't like the username and password supplied.

I would log back in as the user, let's say it's andrew for this example

```bash
sudo -u andrew psql
\password
```

It will now ask you to enter a password twice, we'll pretend I entered password.

The username and password don't have to be super secure as they're just for your test database. I would also update the name rather than using the variable, just because it feels safer to me.

You'll end up with something like this.

```bash
DATABASE_URL="postgresql://andrew:password@localhost:5432/Forem_development"
```

**Comment by cmgorton** [comment]
Thanks for the help on this @Link2Twenty 💜

**Comment by oyeyipowale** [comment]
Unbeknownst to me, the main cause of the  `failed authentication` issue was that the PostgreSQL was running on a different port as suppose to the conventional `5432` port. Discovered this by running the '\conninfo' after login into the user's psql shell.
```bash
andrew=# \conninfo
You are connected to database "andrew" as user "andrew" via socket in "/var/run/postgresql" at port "5433".
```

**Comment by oyeyipowale** [comment]
Hey @cmgorton @Link2Twenty, `bin/startup`; specifically `foreman start -f Procfile.dev`, throws an error. Not just sidekiq, but also the web instance through webpack and database config.

There was a reoccurring theme. The env variables are being read as a whole, both with inline comments.  As is the case below.
```sh
15:00:54 web.1       | started with pid 3142
15:00:54 webpacker.1 | started with pid 3143
15:00:54 sidekiq.1   | started with pid 3144
15:01:05 sidekiq.1   | 2021-05-06T14:01:05.691Z pid=3144 tid=8zs WARN: ArgumentError: invalid value for Integer(): "5 # Specifies the number of Puma threads"
15:01:05 sidekiq.1   | 2021-05-06T14:01:05.692Z pid=3144 tid=8zs WARN: /home/testuser01/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/sidekiq-6.2.1/lib/sidekiq/cli.rb:246:in `Integer'
15:01:05 sidekiq.1   | /home/testuser01/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/sidekiq-6.2.1/lib/sidekiq/cli.rb:246:in `setup_options'
15:01:05 sidekiq.1   | /home/testuser01/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/sidekiq-6.2.1/lib/sidekiq/cli.rb:24:in `parse'
15:01:05 sidekiq.1   | /home/testuser01/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/sidekiq-6.2.1/bin/sidekiq:27:in `<top (required)>'
15:01:05 sidekiq.1   | /home/testuser01/.rbenv/versions/2.7.2/bin/sidekiq:23:in `load'
15:01:05 sidekiq.1   | /home/testuser01/.rbenv/versions/2.7.2/bin/sidekiq:23:in `<top (required)>'
15:01:05 sidekiq.1   | /home/testuser01/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/bundler-2.2.16/lib/bundler/cli/exec.rb:63:in `load'
15:01:05 sidekiq.1   | /home/testuser01/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/bundler-2.2.16/lib/bundler/cli/exec.rb:63:in `kernel_load'
15:01:05 sidekiq.1   | /home/testuser01/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/bundler-2.2.16/lib/bundler/cli/exec.rb:28:in `run'
15:01:05 sidekiq.1   | /home/testuser01/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/bundler-2.2.16/lib/bundler/cli.rb:494:in `exec'
15:01:05 sidekiq.1   | /home/testuser01/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/bundler-2.2.16/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
15:01:05 sidekiq.1   | /home/testuser01/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/bundler-2.2.16/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
15:01:05 sidekiq.1   | /home/testuser01/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/bundler-2.2.16/lib/bundler/vendor/thor/lib/thor.rb:392:in `dispatch'
15:01:05 sidekiq.1   | /home/testuser01/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/bundler-2.2.16/lib/bundler/cli.rb:30:in `dispatch'
15:01:05 sidekiq.1   | /home/testuser01/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/bundler-2.2.16/lib/bundler/vendor/thor/lib/thor/base.rb:485:in `start'
15:01:05 sidekiq.1   | /home/testuser01/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/bundler-2.2.16/lib/bundler/cli.rb:24:in `start'
15:01:05 sidekiq.1   | /home/testuser01/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/bundler-2.2.16/exe/bundle:49:in `block in <top (required)>'
15:01:05 sidekiq.1   | /home/testuser01/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/bundler-2.2.16/lib/bundler/friendly_errors.rb:130:in `with_friendly_errors'
15:01:05 sidekiq.1   | /home/testuser01/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/bundler-2.2.16/exe/bundle:37:in `<top (required)>'
15:01:05 sidekiq.1   | /home/testuser01/.rbenv/versions/2.7.2/bin/bundle:23:in `load'
15:01:05 sidekiq.1   | /home/testuser01/.rbenv/versions/2.7.2/bin/bundle:23:in `<main>'
15:01:06 sidekiq.1   | exited with code 1
15:01:06 system      | sending SIGTERM to all processes
15:01:06 web.1       | terminated by SIGTERM
15:01:06 webpacker.1 | terminated by SIGTERM

== Command ["foreman start -f Procfile.dev"] failed ==
```
The .env file was create from .env_sample. I think all inline comments and variables should be removed/repositioned to avoid such issues.

**Comment by rhymes** [comment]
@oyeyipowale can you replace your lines:

```shell
RAILS_MAX_THREADS=5 # Specifies the number of Puma threads
WEB_CONCURRENCY=2 # Specifies the number of `workers` to boot in clustered mode.
DATABASE_POOL_SIZE=5 # The number of concurrent connections to Postgres. If unset the value of RAILS_MAX_THREADS will be used in it's place.
```

with just:

```shell
RAILS_MAX_THREADS=5
WEB_CONCURRENCY=2
DATABASE_POOL_SIZE=5
```

I'll send a PR shortly to fix that :)

**Comment by cmgorton** [comment]
Thank you @rhymes !

**Comment by Link2Twenty** [comment]
Oh yeah I had that problem too 😅

**Comment by alicenstar** [comment]
I was having this same issue with the same-line comments in the `.env` file. I fixed that, but then was getting this error: `FATAL:  database "$DATABASE_NAME" does not exist`, so `DATABASE_NAME` wasn't getting translated into the `DATABASE_URL`.

I fixed it by hard-coding the `DATABASE_NAME` into the `DATABASE_URL`, but I'm guessing that syntax must be wrong or something for the nested environment variables. I tried several things (removing quotes, changing `$DATABASE_NAME` to `${DATABASE_NAME}`, and combinations of those), but nothing worked besides hard-coding the name into the url.
